ntoskrnl/se/debug.c

```diff
@@ -0,0 +1,388 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Security subsystem debug routines support
+ * COPYRIGHT:       Copyright 2022 George Bișoc <george.bisoc@reactos.org>
+ */
```

Comment:
Too many spaces. Suggesting a fix based on https://reactos.org/wiki/Coding_Style
```suggestion
/*
 * PROJECT:     ReactOS Kernel
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Security subsystem debug routines support
 * COPYRIGHT:   Copyright 2022 George Bișoc <george.bisoc@reactos.org>
 */
```

Reply:
Doesn't look resolved to me.

Reply:
I know we all hate whitespace comments but... now there are too few spaces :sweat_smile:  There are 2 spaces between words, should be 3:
```
 * COPYRIGHT:  Copyright
```